app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
```

Comment:
When importing multiple classes, it's better to use parentheses for clarity and readability. Consider changing this line to: `from app.errors import (VaccineError, NotWearingMaskError)`.